docs/source/performance/caching.mdx

```diff
@@ -0,0 +1,225 @@
+---
+title: Caching
```

Comment:
Suggest something like "Server-side caching in federated graphs" for just the regular `title`